### PR TITLE
Add username who made the commit - Issue #85

### PR DESCRIPTION
**Pull request by ben-m-lucas**
Implementation for adding the username who made the commit

### CODE REVIEW

**Comment by Haacked** [comment]
Thanks! I'll try and look at this soon.

**Comment by Haacked** [comment]
One of the goals of SeeGit is to be a teaching tool during a live talk. In that scenario, the speaker is the only one making commits so the name is redundant.

My preference would be to add a setting to show the name and default it to be off. We already have other settings. If you make the setting sticky, it should accomplish what you want here.

**Comment by ben-m-lucas** [comment]
I added an option in the settings for showing the Commit Author.

I did have one bit of code in the OnToggleSettings method that I'm not thrilled with.  When the commit author is displayed, it changes the width of each CommitVertex.  When this happens, a call to Relayout needs to be made.  Ideally, the fact that the Graph property of the ViewModel changed should be enough information for the control to know that it needs to lay itself out again.  I'm not familiar enough with the RepositoryGraphLayout control to do this though.

File: SeeGitApp/Extensions/CommitTemplatedAdorner.cs [hunk 1]
```diff
@@ -9,6 +9,7 @@ namespace SeeGit
     /// </summary>
     internal class CommitTemplatedAdorner : Adorner
     {
+        private FrameworkElement _adornedElement;
```
Comment (Haacked) [inline SeeGitApp/Extensions/CommitTemplatedAdorner.cs]: Minor nit, but since this isn't settable, it should be `readonly`.

**Comment by Haacked** [comment]
Maybe the `Width` property doesn't implement `INotifyPropertyChanged` properly. In any case, I'm happy with that trade-off. People don't change their settings all the time.

File: SeeGitApp/Extensions/CommitTemplatedAdorner.cs (hunk 1 above)
Comment (ben-m-lucas) [inline SeeGitApp/Extensions/CommitTemplatedAdorner.cs]: I just committed the change to use readonly.

**Pull request merged**